docs/admin/kubeadm-config.md

```diff
@@ -0,0 +1,37 @@
+---
+approvers:
+- mikedanese
+- luxas
+- errordeveloper
+- jbeda
+title: Kubeadm config
+notitle: true
+---
+{% capture overview %}
+# Kubeadm config
+Beginning with v1.8.0, kubeadm uploads the configuration of your cluster to a ConfigMap called 
+`kubeadm-config` in the `kube-system` namespace , and later reads that configuration when upgrading 
+for a seamless user experience.
```

Comment:
in order to configure system components correctly. This provides a seamless user experience.